### PR TITLE
Use the same icon for clipboard button that crates.io itself uses

### PR DESCRIPTION
**Pull request by notriddle**
image

### CODE REVIEW

**Comment by jyn514** [comment]
Hmm, I'm not a giant fan of the icon rustdoc uses - maybe we could make it a clipboard for both instead? https://fontawesome.com/icons/clipboard?style=solid

cc @GuillaumeGomez

**Comment by GuillaumeGomez** [comment]
We don't use fontawesome in rustdoc, so the only thing I can suggest is replacing the rustdoc one with a svg so then we can use the same vector image in docs.rs. How does it sound?

**Comment by jyn514** [comment]
Yes, that sounds good  :) I just think it should be a clipboard icon or something similar.

**Comment by GuillaumeGomez** [comment]
I don't like much the current icon so very open to changes. :)

**Comment by notriddle** [comment]
I don't like the current icon very much either, but I like that it's a unicode character instead of an SVG. Less overhead.

How about `\u{1f4cb}\u{fe0e}`, the [text presentation mode of the clipboard emoji](http://unicode.org/emoji/charts/emoji-variants.html#1f4cb)? 📋︎

**Comment by GuillaumeGomez** [comment]
The current font rendering on mac is quite bad and broken in fact, which is why I suggested an SVG. Simpler to have a unified rendering.

**Comment by jyn514** [comment]
That seems pretty complicated for a button :/ what do you mean by bad and broken?

**Comment by notriddle** [comment]
So, are we going to import an SVG copy icon into rustdoc, or use the `\u{1f4cb}\u{fe0e}` icon in both?

**Comment by GuillaumeGomez** [comment]
I'd prefer an SVG icon like we do for other icons in rustdoc. Also cc @jsha

**Comment by jsha** [comment]
See also https://github.com/rust-lang/rust/issues/84639, where @rukai helpfully posted a screenshot of how crates.io does it. I think it's really valuable to harmonize the icon and the interaction between crates.io, doc.rust-lang.org, and docs.rs. And the crates.io icon is an SVG! 🎉

**Comment by notriddle** [comment]
@jsha @GuillaumeGomez

Okay, here's the crates.io icon.

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
cc @Nemo157 (since it's front-end ;) )

File: templates/header/package_navigation.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h1 id="crate-title">
                         {{ title }}
                     {%- else -%}
                         {{ metadata.name }} {{ metadata.version }}
-                        {{ "copy" | far(id="clipboard", aria_label="Copy crate name and version information", fa=true) }}
+                        <span id="clipboard" class="fa-svg fa-svg-fw" aria-label="Copy crate name and version information">{%- include "clipboard.svg" -%}</span>
```
Comment (Nemo157) [inline templates/header/package_navigation.html:30]: We probably want this label as a title? As mentioned on <https://github.com/rust-lang/rust/issues/84639>/<https://github.com/rust-lang/rust/issues/84986> it's not obvious what this button does. (I personally don't know enough about ARIA to know whether the `aria-label` would still be necessary as well or not).

Comment (notriddle) [reply templates/header/package_navigation.html:30]: Okay, that's done.

**Pull request merged**